Binary files chasen_orig/chasen-2.4.5/debian/.swp and chasen_26/chasen-2.4.5/debian/.swp differ
diff -upNr chasen_orig/chasen-2.4.5/debian/libchasen2.symbols.mips chasen_26/chasen-2.4.5/debian/libchasen2.symbols.mips
--- chasen_orig/chasen-2.4.5/debian/libchasen2.symbols.mips	2016-08-23 06:16:10.000000000 +0000
+++ chasen_26/chasen-2.4.5/debian/libchasen2.symbols.mips	2016-09-16 09:09:51.000000000 +0000
@@ -30,8 +30,11 @@ libchasen.so.2 libchasen2 #MINVER#
  _ZN5Darts15DoubleArrayImplIchlmNS_6LengthIcEEED0Ev@Base 2.4.5
  _ZN5Darts15DoubleArrayImplIchlmNS_6LengthIcEEED1Ev@Base 2.4.5
  _ZN5Darts15DoubleArrayImplIchlmNS_6LengthIcEEED2Ev@Base 2.4.5
- _ZNSt6vectorIN5Darts15DoubleArrayImplIchlmNS0_6LengthIcEEE6node_tESaIS5_EE13_M_insert_auxEN9__gnu_cxx17__normal_iteratorIPS5_S7_EERKS5_@Base 2.4.5
- _ZNSt6vectorIlSaIlEE13_M_insert_auxEN9__gnu_cxx17__normal_iteratorIPlS1_EERKl@Base 2.4.5
+ _ZNKSt5ctypeIcE8do_widenEc@Base 2.4.5-26
+#MISSING: 2.4.5-26# _ZNSt6vectorIN5Darts15DoubleArrayImplIchlmNS0_6LengthIcEEE6node_tESaIS5_EE13_M_insert_auxEN9__gnu_cxx17__normal_iteratorIPS5_S7_EERKS5_@Base 2.4.5
+ _ZNSt6vectorIN5Darts15DoubleArrayImplIchlmNS0_6LengthIcEEE6node_tESaIS5_EE19_M_emplace_back_auxIJRKS5_EEEvDpOT_@Base 2.4.5-26
+#MISSING: 2.4.5-26# _ZNSt6vectorIlSaIlEE13_M_insert_auxEN9__gnu_cxx17__normal_iteratorIPlS1_EERKl@Base 2.4.5
+ _ZNSt6vectorIlSaIlEE19_M_emplace_back_auxIJRKlEEEvDpOT_@Base 2.4.5-26
 #MISSING: 2.4.5-10# _ZNSt8_Rb_treeISsSt4pairIKSslESt10_Select1stIS2_ESt4lessISsESaIS2_EE10_M_insert_EPKSt18_Rb_tree_node_baseSB_RKS2_@Base 2.4.5
 #MISSING: 2.4.5# _ZNSt8_Rb_treeISsSt4pairIKSslESt10_Select1stIS2_ESt4lessISsESaIS2_EE15_M_insert_equalERKS2_@Base 2.4.5
  _ZTIN5Darts15DoubleArrayImplIchlmNS_6LengthIcEEEE@Base 2.4.5
diff -upNr chasen_orig/chasen-2.4.5/debian/libchasen2.symbols.mipsel chasen_26/chasen-2.4.5/debian/libchasen2.symbols.mipsel
--- chasen_orig/chasen-2.4.5/debian/libchasen2.symbols.mipsel	2016-08-23 06:16:10.000000000 +0000
+++ chasen_26/chasen-2.4.5/debian/libchasen2.symbols.mipsel	2016-09-16 10:57:21.000000000 +0000
@@ -30,8 +30,13 @@ libchasen.so.2 libchasen2 #MINVER#
  _ZN5Darts15DoubleArrayImplIchlmNS_6LengthIcEEED0Ev@Base 2.4.5
  _ZN5Darts15DoubleArrayImplIchlmNS_6LengthIcEEED1Ev@Base 2.4.5
  _ZN5Darts15DoubleArrayImplIchlmNS_6LengthIcEEED2Ev@Base 2.4.5
- _ZNSt6vectorIN5Darts15DoubleArrayImplIchlmNS0_6LengthIcEEE6node_tESaIS5_EE13_M_insert_auxEN9__gnu_cxx17__normal_iteratorIPS5_S7_EERKS5_@Base 2.4.5
- _ZNSt6vectorIlSaIlEE13_M_insert_auxEN9__gnu_cxx17__normal_iteratorIPlS1_EERKl@Base 2.4.5
+#_ZNSt6vectorIN5Darts15DoubleArrayImplIchlmNS0_6LengthIcEEE6node_tESaIS5_EE13_M_insert_auxEN9__gnu_cxx17__normal_iteratorIPS5_S7_EERKS5_@Base 2.4.5
+#_ZNSt6vectorIlSaIlEE13_M_insert_auxEN9__gnu_cxx17__normal_iteratorIPlS1_EERKl@Base 2.4.5
+ _ZNKSt5ctypeIcE8do_widenEc@Base 2.4.5-26
+#MISSING: 2.4.5-26# _ZNSt6vectorIN5Darts15DoubleArrayImplIchlmNS0_6LengthIcEEE6node_tESaIS5_EE13_M_insert_auxEN9__gnu_cxx17__normal_iteratorIPS5_S7_EERKS5_@Base 2.4.5
+ _ZNSt6vectorIN5Darts15DoubleArrayImplIchlmNS0_6LengthIcEEE6node_tESaIS5_EE19_M_emplace_back_auxIJRKS5_EEEvDpOT_@Base 2.4.5-26
+#MISSING: 2.4.5-26# _ZNSt6vectorIlSaIlEE13_M_insert_auxEN9__gnu_cxx17__normal_iteratorIPlS1_EERKl@Base 2.4.5
+ _ZNSt6vectorIlSaIlEE19_M_emplace_back_auxIJRKlEEEvDpOT_@Base 2.4.5-26
 #MISSING: 2.4.5-10# _ZNSt8_Rb_treeISsSt4pairIKSslESt10_Select1stIS2_ESt4lessISsESaIS2_EE10_M_insert_EPKSt18_Rb_tree_node_baseSB_RKS2_@Base 2.4.5
 #MISSING: 2.4.5# _ZNSt8_Rb_treeISsSt4pairIKSslESt10_Select1stIS2_ESt4lessISsESaIS2_EE15_M_insert_equalERKS2_@Base 2.4.5
  _ZTIN5Darts15DoubleArrayImplIchlmNS_6LengthIcEEEE@Base 2.4.5
